lms/static/scripts/frontend_apps/components/EmailNotificationsPreferences.tsx

```diff
@@ -6,13 +6,13 @@ import { useCallback } from 'preact/hooks';
 import type { EmailNotificationsPreferences, WeekDay } from '../config';
 
 const dayNames: [WeekDay, string][] = [
-  ['instructor_email_digests.days.sun', 'Sunday'],
-  ['instructor_email_digests.days.mon', 'Monday'],
-  ['instructor_email_digests.days.tue', 'Tuesday'],
-  ['instructor_email_digests.days.wed', 'Wednesday'],
-  ['instructor_email_digests.days.thu', 'Thursday'],
-  ['instructor_email_digests.days.fri', 'Friday'],
-  ['instructor_email_digests.days.sat', 'Saturday'],
+  ['sun', 'Sunday'],
+  ['mon', 'Monday'],
+  ['tue', 'Tuesday'],
+  ['wed', 'Wednesday'],
+  ['thu', 'Thursday'],
+  ['fri', 'Friday'],
+  ['sat', 'Saturday'],
```

Comment:
It turns out that in the end, the BE does not expose the `instructor_email_digests.days.` prefix, which is nicer.

I missed the discussion that happened in the original PR around not exposing this.